doc/source/serve/monitoring.md

```diff
@@ -233,15 +233,100 @@ In addition to the standard Python logger, Serve supports custom logging. Custom
+:end-before:  __level_end__
+:language: python
+```
+
+The `Model` replica log file should also print `DEBUG` level message.
+
+```
+INFO 2023-12-07 13:07:15,788 Model default#Model#LMqLpq f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1 / default replica.py:719 - Started executing request f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1
+DEBUG 2023-12-07 13:07:15,788 Model default#Model#LMqLpq f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1 / default logging_config.py:26 - this is debug message
+INFO 2023-12-07 13:07:15,788 Model default#Model#LMqLpq f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1 / default replica.py:745 - __CALL__ OK 0.1ms
+```
+
+#### Disable access log
+You can also disable access log by passing `disable_access_log=True` to `logging_config` argument `@serve.deployment`. For example
+
+```{literalinclude} doc_code/monitoring/logging_config.py
+:start-after: __enable_access_log_start__
+:end-before:  __enable_access_log_end__
+:language: python
+```
+
+The `Model` replica log file doesn't include serve traffic log, you should only see your application log in the log file.
+
+```
+INFO 2023-12-07 13:15:07,979 Model default#Model#LhNrRV 8c5fcdb2-87a0-46ed-b2f8-9336280c9c3d / default logging_config.py:50 - hello world
+```
+
+#### Configure logging in different deployments and applications
+You can also configure logging at the application level by passing `logging_config` to `serve.run`. For example
+
+```{literalinclude} doc_code/monitoring/logging_config.py
+:start-after: __application_and_deployment_start__
+:end-before:  __application_and_deployment_end__
+:language: python
+```
+
+In Router log file, you should see
+
+```
+INFO 2023-12-07 14:18:18,811 Router default#Router#iNdKWL 6fe398fa-17d6-4abe-8264-d69c12aa9884 / default replica.py:719 - Started executing request 6fe398fa-17d6-4abe-8264-d69c12aa9884
+DEBUG 2023-12-07 14:18:18,811 Router default#Router#iNdKWL 6fe398fa-17d6-4abe-8264-d69c12aa9884 / default logging_config.py:68 - this is debug message from router
+```
+
+In the Model log file, you should see
+
+```
+INFO 2023-12-07 14:18:18,821 Model default#Model#AHYvjY 6fe398fa-17d6-4abe-8264-d69c12aa9884 / default replica.py:719 - Started executing request 6fe398fa-17d6-4abe-8264-d69c12aa9884
+DEBUG 2023-12-07 14:18:18,821 Model default#Model#AHYvjY 6fe398fa-17d6-4abe-8264-d69c12aa9884 / default logging_config.py:75 - this is debug message from model
+INFO 2023-12-07 14:18:18,821 Model default#Model#AHYvjY 6fe398fa-17d6-4abe-8264-d69c12aa9884 / default replica.py:745 - __CALL__ OK 0.1ms
+```
+
+When you set `logging_config` at the application level, it will be applied to all deployments in the application. When you set `logging_config` at the deployment level at the same time, the deployment level configuration will override the application level configuration.
+
+#### Configure logging for serve components
+You can also update logging configuration similar above to serve components by passing `system_logging_config` to `serve.start`.
```

Comment:
specify specifically that this applies to the controller and proxy

Reply:
replace "serve components" to "serve controller and proxies".